pyanalyze/test_name_check_visitor.py

```diff
@@ -1663,6 +1663,21 @@ def capybara(condition):
             print(x)  # static analysis: ignore[undefined_name]
 
 
+class TestNestedLoop(TestNameCheckVisitorBase):
+    @assert_passes()
+    def test(self):
+        def capybara(x: int):
+            v = 1
+            while x < 2:
+                while True:
+                    if x == 0:
+                        assert_is_value(
+                            v, MultiValuedValue([KnownValue(1), KnownValue(2)])
```

Comment:
```suggestion
                            v, KnownValue(1) | KnownValue(2)
```
More concise syntax (probably will also need to reformat).